person.rb

```diff
@@ -1,15 +1,16 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :id, :name
+  attr_reader :rentals, :name
 
```

Comment:
- [ ] Great work so far but it would be nice if you make the getter and setter for name, rentals and age means your code should look like ⬇️ to follow the project requir5ements as previous reviewer suggested 👍

```ruby
  attr_accessor :rentals, :name, :age
```